sycl/include/sycl/device_selector.hpp

```diff
@@ -91,6 +91,14 @@ class __SYCL_EXPORT host_selector : public device_selector {
   int operator()(const device &dev) const override;
 };
 
+// -------------- SYCL 2020
+
+// SYCL 2020 standalone selectors
+__SYCL_EXPORT int default_selector_v(const device &dev);
+__SYCL_EXPORT int gpu_selector_v(const device &dev);
+__SYCL_EXPORT int cpu_selector_v(const device &dev);
+__SYCL_EXPORT int accelerator_selector_v(const device &dev);
+
```

Comment:
I think we should move it to the top of the file (as that's the most recent/modern interface and we want reader to get familiar with state of the art, not deprecated ways). After that, the section with old-style selectors above should be prefixed with a comment saying that they're remnants of SYCL 1.2.